win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5471,8 +5534,8 @@ GreExtTextOutW(
 
             DestRect.left = BackgroundLeft;
             DestRect.right = (TextLeft + (realglyph->root.advance.x >> 10) + 32) >> 6;
-            DestRect.top = TextTop + yoff - ((fixAscender + 32) >> 6);
-            DestRect.bottom = TextTop + yoff + ((32 - fixDescender) >> 6);
+            DestRect.top = TextTop;
+            DestRect.bottom = TextTop + ((fixAscender + fixDescender) >> 6);
```

Comment:
Can you check whether this change is the one that causes the shift-down-by-1-pixel observed in some of the screenshots you provided?

Reply:
This was correct.
A test program: https://jira.reactos.org/secure/attachment/47780/FontPixels.zip

CURRENTLY:
![font-pixel-before](https://user-images.githubusercontent.com/2107452/43436769-dbcc18aa-94c0-11e8-9bb5-786c2bc224db.png)

REVERTED:
![font-pixel-after](https://user-images.githubusercontent.com/2107452/43436780-e6fed47e-94c0-11e8-86c9-fad8312b2bbc.png)